src/styles.scss

```diff
@@ -6,10 +6,11 @@
 @import "src/_sass/layout";
 @import "src/_sass/masthead";
 @import "src/_sass/posts";
+@import "src/_sass/tables";
```

Comment:
I think you forgot to stage/commit this change, and for that reason the styling (on the preview url) is gone.

Reply:
My first commit disappeared due to no-write access. I think this caused it, I'll look into that.

Reply:
Seems to be fine now.